### DISCUSSION TITLE
Email Not Accepted? This teacher needs help!

### DISCUSSION THREAD

**Discussion by tammiramsey** [community/community#102877] [GitHub Education]
### Hi everyone,

Hey everyone - 

I have been trying to get my classroom set up and confirm myself as a teacher. However, my email ends in @Xschools.org and not @Xstudents.org so I can't move on in the application. I opened a help ticket a month ago (on 1/1) and still haven't gotten a response from the support team. Any thoughts on what I could do? This is really frustrating and basically meant I've had to rearrange my semester plans for my intro class in two days. Help!

**Comment by phuccoder2000** [comment]


**Reply by tammiramsey** [reply]
I can't even get that far. I edited the screengrab of the error message I get.
![Screenshot 2024-01-01 at 10 31 16 PM](https://github.com/community/community/assets/97861522/153f006e-c9da-4bd0-a007-92141f3eaf3b)
